package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "toggle-youtube-comments",
+  "version": "2.5.2",
+  "description": "Hides YouTube comments by default",
+  "main": "script.js",
```

Comment:
I forgot to change it since created with `npm init`. Thanks for pointing out.
However, since this "main" key is for the node-module package, so no problem even if it erases. What do you think?

Reply:
Yeah let's get rid of it if it's not needed.